internal/myks/assets/data-schema.ytt.yaml

```
@@ -112,6 +112,11 @@ yttPkg:
 render:
   #! If true, the render output file names will include the namespace
   includeNamespace: false
+#! Configuration options for caching during vendir sync step.
+cache:
+  #! If true, the cache is enabled for all apps per default and can be selectively disabled with the vendir lazy flag.
+  #! If false, the cache is disabled for all apps per default and can be selectively enabled with the vendir lazy flag.
+  enabled: true
```

What do you think about moving this under the `sync` "namespace"?

```suggestion
#! Configuration options for the sync step.
sync:
  cache:
    #! If true, the cache is enabled for all apps per default and can be selectively disabled with the vendir lazy flag.
    #! If false, the cache is disabled for all apps per default and can be selectively enabled with the vendir lazy flag.
    enabled: true
```